system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -85,6 +85,18 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
    */
   virtual system_metrics_collector::ProcCpuData makeSingleMeasurement();
 
+  /**
+   * This publishes the statistics derived from the collected measurements
```

Comment:
Please use imperative tone to follow the Google C++ style: `These comments should open with descriptive verbs in the indicative mood ("Opens the file") rather than verbs in the imperative ("Open the file")`

https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
Done.